caput/memh5.py

```diff
@@ -2558,14 +2564,16 @@ def deep_group_copy(
         names of all datasets that were skipped. If `False` (default) throw a
         `ValueError` if any distributed datasets are encountered.
     postprocess : function, optional
-        A function that takes is called on each node, with the source and destination
+        A function which is called on each node, with the source and destination
         entries, and can modify either.
-    deep_copy_dsets : bool, optional
-        Explicitly deep copy all datasets. This will only alter behaviour when copying
-        from memory to memory. XXX: enabling this in places where it is not currently
-        enabled could break legacy code, so be very careful
-    shared : list, optional
-        List of datasets to share, if `deep_copy_dsets` is True. Otherwise, no effect.
+    shallow : bool, optional
+        Explicitly share all datasets. This will only alter behaviour when copying
+        from memory to memory. If False, any dataset listed in `shared` will NOT be copied.
+        Default is False.
+    shared : iterable, optional
+        Iterable (list, set, generator) of datasets to share, if `deep_copy_dsets` is True.
+        Otherwise, no effect. Shared datasets just point to the existing object in g1 storage.
+        Axis selections cannot be applied to shared datasets.
```

Comment:
```suggestion
    shared : iterable, optional
        Iterable (list, set, generator) of datasets to share, if `shallow` is False. Shared
        datasets just point to the existing object in g1 storage.  Axis selections cannot
        be applied to shared datasets.    Ignored if `shallow` is True, since, in that case,
        _all_ datasets are shared.
```